pKa_challenge_instructions.md

```diff
@@ -0,0 +1,115 @@
+Please do not create a microstate ID yourself. It is important that challenge organizers assign unique microstate IDs and keep track.  
+
+## Submission of multiple predictions
+
+Some participants use SAMPL to help evaluate various computational methods. To accommodate this, multiple prediction sets from a single research group or company are allowed, even for same type of predictions if they are made by different methods.
+
+## Uploading your predictions
+
+D3R is currently outfitting the SAMPL6 page with the ability to accept your uploaded predictions. As soon as this is ready, you may upload your predictions. If you want to upload more than one type predictions (see description of three pKa prediction types above) or different set of predictions (same type but generated by different methods), each must be uploaded as a separate file. Please use the template provided, as the predictions will be parsed and analyzed with automated scripts. Please include predictions related all molecules with same method and same submission type in one file. 
+
+We encourage submitting predictions in all three formats and for all 24 molecules when possible. Incomplete submissions - such as for a subset of compounds - will also be accepted, but will not necessarily be evaluated together with the rest of the submissions. However, we would emphasize that omission of SEM estimates will not cause a submission to be regarded as incomplete.
+
+Names of the prediction files must begin with the name of the submission type (i.e., typeIII), and your name and must end with an integer indicating set of prediction. For example, if you want to submit one prediction file for type III prediction (macroscopic pKas), you might name it `typeIII-myname-1.csv`, where myname is arbitrary text of your choice. If you submit two prediction files of the same submission type, you might name them `typeIII-myname-1.txt` and `typeIII-myname-2.txt`.
+
+The file will be machine parsed, so correct formatting is essential.
+
+Lines beginning with a hash-tag (#) may be included as comments. These and blank lines will be ignored.
+
+The file must contain the following four components in the following order: your predictions, a name for your computational protocol, a list of the major software packages used, and a long-form methods description. Each of these components must begin with a line containing only the corresponding keyword: `Predictions:`, `Name:`, `Software:`, and `Method:`, as illustrated in the example files. More detailed instructions will be provided on the challenge submission site on the D3R website.  
+
+## Evaluation strategy for computational pKa predictions
+
+Macroscopic pKa predictions will be directly compared to experimental pKa measurements. Microscopic pKas and predicted microstates will be evaluated by comparison to each other and how well they recapitulate experimental macroscopic pKas, as well as pKa predictions of ACD which uses a data-trained pKa predictions method.
```

Comment:
"data-trained pKa prediction method". Also, this statement sounds rather broad. Do you mean an empirical one? Most pKa methods involve some kind of data, I would imagine.

Reply:
Yes, empirical is the right terminology here. I corrected it.